pkg/registry/flowcontrol/ensurer/strategy.go

```diff
@@ -113,37 +95,70 @@ type configurationObject interface {
 	runtime.Object
 }
 
+type configurationObjectType interface {
+	comparable
+	configurationObject
+}
+
+type objectOps[ObjectType configurationObjectType] struct {
+	Client[ObjectType]
+	Cache[ObjectType]
+	deepCopy    func(ObjectType) ObjectType
+	replaceSpec func(ObjectType, ObjectType) ObjectType
+	specEqual   func(expected, actual ObjectType) bool
+}
+
+func NewObjectOps[ObjectType configurationObjectType](client Client[ObjectType], cache Cache[ObjectType],
+	DeepCopy func(ObjectType) ObjectType,
```

Comment:
function args are not usually `PublicStyle` but `privateStyle` names

Reply:
Thanks, fixed.